README.md

```diff
@@ -15,15 +15,16 @@ log collection.
 
 ### Submitting patches
 
-Please use GitHub pull requests.
+Please use [GitHub pull requests](https://github.com/oVirt/vdsm/pulls).
+See [Development environment setup](#development-environment-setup) for more details.
```

Comment:
Link is useful, but we want to move the section to a new document, so better wait until this work is done.